rules/prettier.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  rules: {
+    'prettier/prettier': [
+      'error',
+      {
+        proseWrap: 'always',
+        singleQuote: true,
+        trailingComma: 'all',
```

Comment:
We should specify `arrowParens`: true, as it's always necessary when specifying return types in TypeScript, which we require.

We should also specify `endOfLine` as `lf`, so it's automatically fixed for Windows users.